app/controllers/new_administrateur/groupe_instructeurs_controller.rb

```diff
@@ -158,6 +159,24 @@ def update_routing_criteria_name
         notice: "Le libellé est maintenant « #{procedure.routing_criteria_name} »."
     end
 
+    def import
+      file = params[:file]
```

Comment:
```suggestion
      group_csv_file = params[:group_csv_file]
```

J'ai pas mal peur d'une attaque par un fichier pourri, est ce que tu peux checker :
- le content-type est bien un csv en checkant `content-type` + en utilisant https://github.com/rails/marcel ?
- que la taille `.size` fait moins d'un 1Go ?


Et en plus, on pourrait peut être faire un check des colonnes et renvoyer une jolie erreur s'il y a pas `groupe` et `email`